docs/units/logarithmic_units.rst

```diff
@@ -137,6 +137,35 @@ appropriate equivalency::
     >>> V.to(u.ABmag, u.spectral_density(5500.*u.AA))  # doctest: +FLOAT_CMP
     <Magnitude [16.99023831, 21.10978201, 22.50053827] mag(AB)>
 
+This is particularly useful for converting magnitude into flux density.
+``V`` is currently in ST magnitudes, which is based on flux densities per
+unit wavelength::
```

Comment:
To covert ``V`` into a flux density per wavelength, ``f_lam``::

Reply:
```suggestion
unit wavelength (:math:`f_\lambda`). Therefore, we can directly convert ``V`` into flux density per unit wavelength using the :meth:`~astropy.units.quantity.Quantity.to` method::
```